README.md

```diff
@@ -78,7 +78,7 @@ A curated list of awesome resources related to *Soccer Analytics* in *english* a
 
 ## Data
 ### Datasets
-* [A public data set of spatio-temporal match events in soccer competitions]
+* [A public data set of spatio-temporal match events in soccer competitions](https://doi.org/11.6084/m9.figshare.c.4415000) : Free - .json - UEFA Euro Cup 2016 - La Liga, Serie A, Bundesliga, Premier League, Ligue 1, FIFA World Cup 2018
```

Comment:
The link doesn't work.

Reply:
Sorry, try again: 

https://figshare.com/collections/Soccer_match_event_dataset/4415000